examples/jf-control-app/commands/example/ExampleOperationalCredentialsIssuer.cpp

```diff
@@ -457,6 +457,70 @@ ExampleOperationalCredentialsIssuer::GenerateNOCChain(const ByteSpan & csrElemen
     return CHIP_NO_ERROR;
 }
 
+CHIP_ERROR ExampleOperationalCredentialsIssuer::SignICAC(const ByteSpan & icaCsr, MutableByteSpan & icac)
+{
+    VerifyOrReturnError(mInitialized, CHIP_ERROR_UNINITIALIZED);
+
+    chip::Platform::ScopedMemoryBuffer<uint8_t> rcac;
+    VerifyOrReturnError(rcac.Alloc(kMaxDERCertLength), CHIP_ERROR_NO_MEMORY);
+    MutableByteSpan rcacSpan(rcac.Get(), kMaxDERCertLength);
+
+    ChipDN rcac_dn;
+    CHIP_ERROR err = CHIP_NO_ERROR;
+
+    P256PublicKey pubkey;
+    ReturnErrorOnFailure(VerifyCertificateSigningRequest(icaCsr.data(), icaCsr.size(), pubkey));
+
+    uint16_t rcacBufLen = static_cast<uint16_t>(std::min(rcacSpan.size(), static_cast<size_t>(UINT16_MAX)));
+    PERSISTENT_KEY_OP(mIndex, kOperationalCredentialsRootCertificateStorage, key,
+                      err = mStorage->SyncGetKeyValue(key, rcacSpan.data(), rcacBufLen));
+    ReturnErrorOnFailure(err);
+
+    {
+        uint64_t rcacId;
+        // Found root certificate in the storage.
+        rcacSpan.reduce_size(rcacBufLen);
+        ReturnErrorOnFailure(ExtractSubjectDNFromX509Cert(rcacSpan, rcac_dn));
+        ReturnErrorOnFailure(rcac_dn.GetCertChipId(rcacId));
+        VerifyOrReturnError(rcacId == mIssuerId, CHIP_ERROR_INTERNAL);
+    }
+
+    ChipDN icac_dn;
+    ReturnErrorOnFailure(icac_dn.AddAttribute_MatterICACId(mIntermediateIssuerId));
+
+    ChipLogProgress(Controller, "Signing ICA CSR");
+
+    ReturnErrorOnFailure(
+        IssueX509Cert(mNow, mValidity, rcac_dn, icac_dn, CertType::kIcac, mUseMaximallySizedCerts, pubkey, mIssuer, icac));
+
+    VerifyOrReturnError(CanCastTo<uint16_t>(icac.size()), CHIP_ERROR_INTERNAL);
+
+    return CHIP_NO_ERROR;
+}
+
+CHIP_ERROR ExampleOperationalCredentialsIssuer::SignNOC(const ByteSpan & icac, const ByteSpan & nocCsr, MutableByteSpan & noc)
```

Comment:
This method is supposed to be called over RPC when jf-admin-app from Ecosystem B starts to issue new NOCs for the devices from Fabric B after JCM (see [guide](https://github.com/project-chip/connectedhomeip/blob/master/docs/guides/joint_fabric_guide.md))?